src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/project.json

```diff
@@ -1,8 +1,4 @@
 {
-  "supports": {
```

Comment:
I don't know what this is supposed to. Currently NuGet reports warnings which tells me that this is useless.